src/generateRoutesWithHerbarium.js

```diff
@@ -0,0 +1,41 @@
+const generateRoutes = require('./generateRoutes')
+
+function findEntitiesAndGroups(herbarium) {
+    const items = Array.from(herbarium.usecases.all.values()).map(e => ({ id: e.entity, group: e.group }))
```

Comment:
could you use explicit name declaration in `e` variable?
following the clean code, in this way we could to build a easy code to read